lms/djangoapps/courseware/tests/test_access.py

```diff
@@ -123,6 +123,19 @@ def test_has_ccx_coach_role(self):
         self.setup_user()
         self.assertFalse(access.has_ccx_coach_role(self.user, ccx_locator))
 
+    def test_ccx_coach_has_staff_role(self):
+        """
+        Assert that user has staff access on ccx.
+        """
+        ccx_locator = self.make_ccx()
+
+        # coach user has access as staff on ccx
+        self.assertTrue(access.has_access(self.coach, 'staff', ccx_locator))
+
+        # basic user doesn't have access as staff on ccx
+        self.setup_user()
```

Comment:
Can you also give the new user a Staff role on the base course? As it stands, it looks like this test will pass with the previous code as well?